packages/shell-chrome/views/panel.edge

```diff
@@ -12,7 +12,7 @@
         </style>
     </head>
 
-    <body class="antialiased">
+    <body id="devtools-container" class="antialiased">
```

Comment:
@HugoDF It doesn't seem like this is hurting anything, but is there something I'm maybe missing?

Reply:
there isn't but it might work out better to use testid's on the "root panel components" we want to inject in the simulator (see https://github.com/alpine-collective/alpinejs-devtools/pull/136#discussion_r547253321)

Reply:
Just realized the map is only run in the simulator anyway, so it's not necessary here regardless.